dir.props

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+
+  <Import Project="dependencyversions.props" />
+
+  <!-- Branch Info -->
+  <PropertyGroup>
+    <MajorVersion>2</MajorVersion>
+    <MinorVersion>0</MinorVersion>
+    <PatchVersion>0</PatchVersion>
+    <StabilizePackageVersion>false</StabilizePackageVersion>
+    <LockHostVersion>false</LockHostVersion>
+    <ReleaseSuffix>beta</ReleaseSuffix>
+    <Channel>master</Channel>
+    <BranchName>master</BranchName>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
+  </PropertyGroup>
+
+  <!-- Versioning -->
+  <PropertyGroup>
+    <CommitCountString>009999</CommitCountString>
+    <SimpleVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion).$(CommitCountString)</SimpleVersion>
+    <VersionSuffix>$(ReleaseSuffix)-$(CommitCountString)</VersionSuffix>
+    <NuGetVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)-$(VersionSuffix)</NuGetVersion>
+    <HostVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)-$(VersionSuffix)-00</HostVersion>
+    <NetCoreAppVersion>$(HostVersion)</NetCoreAppVersion>
+    <ProductionVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)</ProductionVersion>
+    <SharedFrameworkNugetVersion Condition="'$(StabilizePackageVersion)' == 'true'">$(NetCoreAppVersion)</SharedFrameworkNugetVersion>
+    <SharedFrameworkNugetVersion Condition="'$(StabilizePackageVersion)' == 'false'">$(ProductionVersion)</SharedFrameworkNugetVersion>
+  </PropertyGroup>
+
+  <!--
+    $(OS) is set to Unix/Windows_NT. This comes from an environment variable on Windows and MSBuild on Unix.
+  -->
+  <PropertyGroup>
+    <!-- Temp change to make OS X build behave as a Linux build -->
+    <OsEnvironment Condition="'$(OsEnvironment)'=='' AND '$(OS)'=='OSX'">Unix</OsEnvironment>
+    <OsEnvironment Condition="'$(OsEnvironment)'==''">$(OS)</OsEnvironment>
+  </PropertyGroup>
+
+  <!-- Common repo directories -->
+  <PropertyGroup>
+    <ProjectDir>$(MSBuildThisFileDirectory)</ProjectDir>
```

Comment:
I think `RepoRoot` is a better name for this property.  Using a property named `ProjectDir` in a .csproj is confusing because the "project" is the .csproj.